### ISSUE TITLE
Add protein schema that is useful for fitness / fasta / bioinformatics analysis

### ISSUE THREAD

**Issue by apayne97** [choderalab/asapdiscovery#855]
Particularly for the viral proteins, there is nothing close to a 1:1 correlation between the "Target" (the molecular assembly that is the goal of inhibition) and the "Protein" (the sequence of amino acids of a single polypeptide chain) and the uniprot id. For instance, if you search for the Uniprot ID of MERS Mpro, you get the whole ORF1a.

In order to handle this I think we need to flesh out our schema a bit more. I started trying to do this a bit in this notebook:
https://github.com/choderalab/avoidome-analysis/blob/main/20240207_download_PDB_from_uniprot.ipynb

@hmacdope @JenkeScheen thoughts?

**Comment by JenkeScheen** [comment]
can you push the notebook with cell outputs displayed plz?

**Comment by apayne97** [comment]
done!

**Comment by hmacdope** [comment]
Yes I think we want to think about what the reasonable boundary is between information we want and don't want and also what the hierarchy of schema is. 

Happy to whiteboard this out as this also touches on the issue of defining a target-wide schema  that incorporates all the info one may need to comp-chem on a particular target and making the repo more target independent. 

May I suggest we prototype in avoidome analysis rather than upstream here? That way we can see what works and what doesn't. If you are importing ASAPDiscovery's schema directly you can just subclass it and experiment with more functionality. Happy to discuss @apayne97

**Comment by hmacdope** [comment]
Also touches on the genetics / fitness informed stuff so this info will for sure be very useful.
